src/libraries/tests.proj

```diff
@@ -177,6 +177,72 @@
     <ProjectExclusions Include="$(RepoRoot)\src\tests\FunctionalTests\Android\Device_Emulator\AOT\Android.Device_Emulator.Aot.Test.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetOS)' == 'Android' and '$(EnableTrimming)' == 'true'">
```

Comment:
Can we add `'$(RunDisabledAndroidTests)' != 'true'` to the condition here so we can see if these still fail on a draft PR that runs all the tests?

Reply:
Done, thanks